### DISCUSSION TITLE
access to collection and sub collections

### DISCUSSION THREAD

**Discussion by sadden3194** [dani-garcia/vaultwarden#4444] [General]
Hi all!

I have a collection "Main".
In this collection are many sub collections.

Now I would like to give an user read access to Main and all the sub collections.
Is this possible? Or should I have to give access to each sub collection?

I found no way to solve this.

**Comment by BlackDex** [comment]
See: #3814

**Reply by sadden3194** [reply]
ah okay :(
Thank you!
